python/cudf_polars/cudf_polars/experimental/base.py

```diff
@@ -118,8 +118,23 @@ def unique_stats(self, column: str) -> UniqueStats:  # pragma: no cover
         """Return unique-value statistics for a column."""
         raise NotImplementedError("Sub-class must implement unique_stats.")
 
-    def storage_size(self, column: str) -> ColumnStat[int]:
-        """Return the average column size for a single file."""
+    def partial_file_size(
```

Comment:
We didn't **need** to change the name from `storage_size` to `partial_file_size`, but the old name has felt a bit confusing to me for a while now.